ldes-server-application/examples/minimal-config-application.yml

```diff
@@ -2,8 +2,8 @@ spring:
   main:
     allow-bean-definition-overriding: true
 
+host-name: "http://localhost:8080"
 collections:
```

Comment:
This isn't so much an issue on this PR, but a general remark. 
I should expect that all our configuration is bundled under one group. Like we had with Ldes.host-name
So i'd also expect ldes.collections,  ...

Reply:
edit: This config just doesn't follow the test config. In there, there is a ldes-server group identifier defined